### ISSUE TITLE
VizCon planning

### ISSUE THREAD

**Issue by mkborregaard** [JuliaPlots/AbstractPlotting.jl#304]
Hi,
We've got the VizCom coming up a month from now, and we should start to build up the program. There'll be a welcome email coming up with some practical details as well, so this is for the content. I suggest doing it here on github, as discussions here tend to stay a little more on point and hit a crowd a little more focusedly than on discourse. Let's just ignore that this is the AbstractPlotting repo - it's a nice way to keep things together.

cc @kristofferc @tlienart @asinghvi17 @SebastianM-C @piever @logankilpatrick @c42f @daschw @pfitzseb @jkrumbiegel @SimonDanisch @strangeli @jheinen @cbauer @vilim @oxinabox @glennmoy

The idea is that we have 4.5 days together to give a massive push to plotting / visualization in Julia. Many attendees are involved with packages in JuliaPlots but the game is of course open. If a big enough group agree on a single goal - like pushing Makie from "really promising" to "a relevant option as the daily plotting tool of choice for a larger group of people", it should be possible with the right coordination and planning of tasks. So let's start that process.

In practical terms, Thursday to Sunday will have roughly this structure:

- Start at 10.
- Presentations of packages and projects.
- Decision of tasks and formation of groups.
- Group work. 
- Short feedback of results.
- Lunch 13-14, then
- Decision of tasks and formation of groups.
- Group work. 
- Summary and presentations of the day's achievements and suggestions for the day after.
- End at 18, shared dinner.

On Monday, we will have a half day (ending at lunch) for summarizing and planning follow-up steps.

We'd like to have some presentations-for-contributors of packages, and some of the group work can take the form of discussion groups. Here are a number of projects and presentations we've already talked about having in there:

### Projects

- Makie compatibility with RecipesBase recipes (lead Simon)
- Push Plots 1.0 (whatever that entails in terms of closing issues)(lead Daniel + Michael)
- Theming engine for Makie (lead Michael)
- Incorporate MakieLayout into Makie (lead Julius + Simon)
- Address the time-to-first-plot / static compilation (lead ?)

Other projects could include building recipes for those packages that don't have them yet, making StatsPlots recipes for models definable in StatsModels, finalizing the pgfplotsx backend to Plots, making VegaLite consume recipes, etc etc.

### Discussion groups (perhaps plenary, perhaps in groups)

- Can we revamp RecipesBase recipes to be more powerful and simpler (would require LOTS of PRs updating existing recipes)
- Plan: how do we maintain Plots given it has 500 open issues (and growing)?
- What does the Julia ecosystem need to have in 1 year in terms of plotting?
 
### Presentations-for-contributors

- Design of Makie (Simon)
- How do Plots recipes actually work? (Daniel)
- Theming Makie (Michael)

We hope to have a mix of discussions, coding individually, in smaller groups and teams, presentations, and fun!

To ensure progress it would be great to have some themes - it would be great if anyone would feel like leading one of these themes, and being responsible for identifying some issues to work on. These could be 

### Themes
- recipes
- web
- documentation

So, to everyone here, could you report if you have anything in particular you plan to be working on, suggest one or more projects, consider whether you could give a presentation of your favourite package or project, or lead up a theme? This is the kind of coordination that will make sure we all make the most of this opportunity, which essentially arrives extremely rarely in Julia development! Looking forward to hearing your input.

**Comment by mkborregaard** [comment]
Also, @kescobo @sjkelly @mcmcgrath13 (am I missing anyone?) have announced that they would like to join remotely. Sorry for forgetting to tag you above - we'll do our best to give you a feeling of participitating though you'll have to do it virtually :-)

**Comment by davidanthoff** [comment]
I might be able to join remotely for a bit. I’ll be in Munich, so at least time zone wouldn’t be a problem.

**Comment by KristofferC** [comment]
I can talk about PackageCompiler and some of the design choices we made in PGFPlotsX.jl that I think contributes to us having relatively few issues/bug reports.

**Comment by kescobo** [comment]
Given the timezone, I'm not certain how much I'll be able to join real time; do we have any further info about A/V and recording? If it's not possible, I'll at least try to catch up on issues and slack, though I may be doing most of my contributing while you're all out having beers...

On that note - can any locals send me a few good local brews so I can look for local distributors? I figure this is a good excuse to do some drinking at work 🍻

**Comment by asinghvi17** [comment]
@mkborregaard we should probably put down a paper discussion group, even if just for an hour or so.

**Comment by mkborregaard** [comment]
All great ideas - I really like the idea of participating remotely by drinking the local beers :joy:

**Comment by Tokazama** [comment]
I'd love to contribute wherever I can throughout this. My schedule is super unpredictable lately though, so I'll just try to follow along and chime in and contribute when possible.

**Comment by jheinen** [comment]
I will be there on Friday and Saturday. If the schedule allows, I would love to demonstrate the new features of GR and also explain our deployment strategy (CI/CD). This is very important for the interaction with other packages (plots, GRUtils). At the moment I don't know if GR is still relevant for Makie. I have the impression that CairoMakie is quite sufficient. But I am very interested in an integration with current or future recipe systems (AbstractPlotting , etc.).

**Comment by piever** [comment]
There are a couple of topics that I would like to discuss (not sure about the format).

I think that StatsMakie needs a bit of a "push". I could probably start by introducing the StatsMakie design and API (grammar of graphics in Makie). Then it would be great to clean and finalize the implementation. This is closely linked to the recipes discussion: ideally, it should be possible for the StatsMakie pipeline to just return a list of plots with "recommended attributes", such as layout, or axis labeling.

The second topic is the relationship between Makie and Interact. This in turn has two aspects:
- porting Interact to share the same "web stack" as the WebGL version of Makie (WGLMakie)
- doing the same decoupling Makie does for plots, also for widgets: separate the structure and logic of the UI from the rendering, in such a way that it would be possible to switch from web-app to native app just by changing the backend (as it is for plots in Makie already).

The first topic (StatsMakie) should be feasible within the conference. The second topic is more of a long term plan, but I think it's worth discussing here, especially since right now it is a bit difficult in julia to make apps where both the UI and the plot are interactive, and Interact + WGLMakie could potentially address that.

**Comment by asinghvi17** [comment]
@jheinen, at the moment, CairoMakie doesn't support 3D (and has quite a few weird bugs).  If we could get an interactive GR backend up, that would be really lovely!  It would also be nice to support some form of TeX math in Makie (which I think GR does).

**Comment by SimonDanisch** [comment]
I have no personal attachment to cairo, besides that it was easier to get things working for me... If I had some support from @jheinen, that would easily offset that advantage I suppose...

**Comment by asinghvi17** [comment]
We have https://github.com/JuliaPlots/GRMakie.jl which used to at least draw lines before the backend API changes.  Maybe we could use that as a starting point?

**Comment by asinghvi17** [comment]
I've attempted to codify what we know so far about potential projects and talks into the [Vizcon 2 project](https://github.com/orgs/JuliaPlots/projects/2) in JuliaPlots.  The new columns are [Projects](https://github.com/orgs/JuliaPlots/projects/2#column-7980439) and [Talks](https://github.com/orgs/JuliaPlots/projects/2#column-7984231).  

Please feel free to add yourselves to them if you want to lead a project, or give a talk about something!

**Comment by Tokazama** [comment]
@asinghvi17, lots of nice stuff there! The CoordinateTransformations bit looks like a really nice clean one too considering how much it's used other places.

Are there any places where the new plots recipes stuff is being discussed or developed more specifically? I have some ideas but I'm not sure whether they are even applicable because I have now idea what's going on there.

**Comment by asinghvi17** [comment]
I think Simon was going to present a working example of Makie consuming RecipesBase's recipes on the fly at Vizcon, and Julius has a prototype of a new recipe system for Makie in MakieLayout.

**Comment by mcmcgrath13** [comment]
I'll be with @kescobo on the local beer participation train as the time zones are rough.  I'll do my best to keep up with the conversation and follow along.

**Comment by BeastyBlacksmith** [comment]
I can give an overview of the current status of the `pgfplotsx`-Plots baackend

**Comment by asinghvi17** [comment]
@jkrumbiegel would you be able to give a high-level talk on MakieLayout (preferably on the first day) for the integration project?

**Comment by jkrumbiegel** [comment]
Sure :)

**Comment by vilim** [comment]
I will be happy to work on anything, though I have more experience with the Makie than the Plots ecosystem. I will be arriving on Thursday afternoon.

**Comment by kescobo** [comment]
@mcmcgrath13 First we need someone from Berlin to send recommendations... C'mon people, don't make me do my own research!

**Comment by BeastyBlacksmith** [comment]
Well there are the big brands "Berliner" and "Berliner Bürgerbräu" and a lot of microbreweries, which I don't know if you can get them abroard like "Quartiermeister", "Vagabund", "Eschenbräu" a minimalistic one called "Bier".

**Comment by asinghvi17** [comment]
@mkborregaard, could you post the details about the seminar room TU Berlin has given us over here?  They seem to have been deleted from Slack.

I was curious about the possibility of streaming presentations live, similarly to how JuliaCon did it.  Would that be possible at all?

**Comment by oxinabox** [comment]
> I was curious about the possibility of streaming presentations live, similarly to how JuliaCon did it. Would that be possible at all?

Live streaming presentations is always either incredibly labour intensive, or incredibly expensive.

**Comment by Tokazama** [comment]
I would be happy if it was just recorded and action points relevant to the workgroups were made available live somewhere for people trying to participate remotely.

**Comment by asinghvi17** [comment]
We could also ask the presenters to capture their screen and audio (from the presenting laptop itself).  That seems like a cheap and easy option, and we can always upload to some platform later.

**Comment by logankilpatrick** [comment]
I’ll try to take care of this ( I have a mic and a camera that I’m bringing) for both recording and remote participants. I will be attempting to film and set up a Plotting/visualization course for Julia Academy which is why I am attending VizCon!

**Comment by asinghvi17** [comment]
Perfect, we can figure something out there itself.  Would JuliaLang be able to upload the videos onto their Youtube channel as well (for public distribution)?  I think they did that for JuMP-dev...

**Comment by j-fu** [comment]
> @mcmcgrath13 First we need someone from Berlin to send recommendations... C'mon people, don't make me do my own research!

The hyperlocal stuff I don't normally drink is Berliner Weisse, see e.g. [here](https://www.beeradvocate.com/beer/profile/2858/430236/) and its on tap in MA...

Just sent an email to register, plan to attend Sat-Mon. Working on this one: https://github.com/j-fu/VTKFig.jl  meant as a showcase what can be done with vtk.

**Comment by mkborregaard** [comment]
@strangeli is responsible for the room booking at Technical University - Lia do you know the answer to Anshul's question?

**Comment by SimonDanisch** [comment]
I started slowly going through issues & managing https://github.com/orgs/JuliaPlots/projects/2!
I'll also add a few project descriptions!

**Comment by strangeli** [comment]
> @mkborregaard, could you post the details about the seminar room TU Berlin has given us over here? They seem to have been deleted from Slack.
> 
> I was curious about the possibility of streaming presentations live, similarly to how JuliaCon did it. Would that be possible at all?

This question? The room is our seminar room (EN223) at the Control Systems Group, it has a beamer and a blackboard (!) and space for around 30 people. There is no special streaming equipment in place and I honestly wouldn't know what equipment is needed for that. I have one paper deadline on Monday which is quite tough for me... :-/ If anyone sends me a list of technical equipment (camera+stand/microphone/???), I could try to find out next week if I can borrow it somewhere. 

My student time is over quite a bit such that I might not be fully up to date but generally the rooms with good equipment for whatever are the large lecture rooms which have these fixed seats that go "uphill" and live-streaming is not a usual case here either. For a technical university, things are still quite analog here or I don't know them, yet :-). I know that they started to film some lectures though so that there must be at least one camera around at that university. I promise, I will at least try to hunt it down. ;-)

Sorry that I don't have a better answer. Looking forward to meeting you anyway.

**Comment by j-fu** [comment]
I can offer a talk on vtk for Sat or Sun, kind of "Whats in vtk which would be nice to have in Makie, with examples". Essentially its scalar and vector fields on 2D and 3D simplex meshes and it would be based on a Julia wrapper around a C wrapper around a C++ wrapper around the vtk API.
May be not exactly what can be done immediately for Makie, but just some ideas for thoughts on directions.

**Comment by mkborregaard** [comment]
Thanks @strangeli . It sounds like @logankilpatrick is bringing a camera, so you shouldn't need to be hunting around for one. Good luck on the paper deadline!

**Comment by oxinabox** [comment]
https://en.wikipedia.org/wiki/Unconference

**Comment by mkborregaard** [comment]
That's an awesome term. VizUnCon!

**Comment by c42f** [comment]
> If a big enough group agree on a single goal - like pushing Makie from "really promising" to "a relevant option as the daily plotting tool of choice for a larger group of people",

Personally I'd love this to be the overall goal, and I'm keen to work on any project which we decide has the best chance of making it happen. Layout and recipes seem to be two key things there.

I also wonder whether there's a way we can move from Plots to Makie via evolution (not revolution, because IMHO that rarely seems to work out very well)? Yes, there's an impedance mismatch, but could we revisit AbstractPlotting as a viable Plots backend?

---

As a user, I really want to use Makie for all the things, but for everyday use I still find myself going back to other systems for various reasons. It's a bit of a death-by-1000-cuts in needing obscure features more often than you might think, but also in not understanding how to fix things and being confused by the documentation.

One thing I'd like to get from this meeting is a really solid understanding of how the Makie ecosystem is put together at a high level such that I can both use it and hack on it effectively.

With that in mind, here are a few things which have confused me in the past, and subjectively led me to feel like I didn't "get" the API in a fundamental way.
* I didn't understand the partitioning of work between AbstractPlotting vs the backends.
* I didn't really understand "what kinds of things exist" in the makie world. That is, I didn't really understand the data model at a high level.
* As a user I didn't understand how to discover keyword arguments, or the system for choosing them in the first place.
* I didn't fully understand the capabilities of the recipe system (even though I wrote some docs for it as I found it six months ago ;-) )
* I found the rules for lifting signals confusing. Various things which I thought I should be able to animate didn't work, but I was confused about how to fix them. I also felt like there was a lot of boilerplate involved in lifting and some confusing terminology. Perhaps the APIs could be improved here.

Now, I haven't spent any time looking at these things for perhaps six months or so — maybe things have changed — but a lot of these issues could be addressed with improved documentation. So a possible project I could lead would just be to learn as much as possible from the experts (that is, ask all the dumb questions!), and to attempt to capture that into both improved user docs and devdocs to help other developers get going.

**Comment by mkborregaard** [comment]
> Personally I'd love this to be the overall goal

I agree that this would be a really good overall goal of the (un-)conference. Still - we should maintain that this is open - everyone should work on what they feel like the most, and anyone who will come with a project and inspire a group of people to help should just go ahead with that. I imagine that some people will be leading ideas and projects, whereas others will mainly help. But agreed, if a large proportion of people want to make this happen, I think that would be a really great outcome.
My main concern is that Simon is going to have a limited bandwidth - so if many people should focus on Makie it requires that the people who know Makie really well think about how they can stimulate this progress.

> could we revisit AbstractPlotting as a viable Plots backend?

I think actually RecipesBase capability of Makie could solve this another way around. Given that (almost) any Plots command can be expressed as a recipe it should be easy to create a link from Plots to Makie once that is in place.

**Comment by mkborregaard** [comment]
Hi all, I've made a very preliminary program - this is just a suggestion to say something about how the days may look, how we can organize the time, etc. Thoughts? How do we collaborate best about this?

<img width="1113" alt="Screenshot 2020-02-22 at 12 09 14" src="https://user-images.githubusercontent.com/8429802/75091342-69518300-556c-11ea-8cfd-7844f13e3424.png">
I also updated the list of attendees: https://gist.github.com/mkborregaard/b396414dc20630d17da193b6db5f084c

**Comment by jkrumbiegel** [comment]
I want to learn more about the possibilities of the different Makie backends. I have a very good mental model of how a library like matplotlib assembles a plot description and how a backend converts it to pdf or png.

But I think because GPU based plotting is more "stateful" (buffers need to be allocated, shaders loaded, drawing is not as sequential but more parallel) I don't understand what we can or can't do, especially because it gets so complicated when we talk about interactive plots in GLMakie, web based stuff in WGLMakie and static plots in Cairomakie all at the same time.

Not understanding how scenes worked inhibited the initial MakieLayout work quite a bit, for example. I honestly still don't even understand how these nested plot objects / recipes work in Makie, but maybe this system will be changed during vizcon anyway. Right now I just see a forest of conversion functions with unforeseen effects :)

So hopefully after VizCon I'll be able to contribute much more efficiently. My goal is to establish a stronger foundation with scene clipping and axes, for example, that support data transformations so we're not so limited to standard 2d and 3d plots anymore in the Makie ecosystem.

**Comment by jkrumbiegel** [comment]
To add to this, I think it's best if we concentrate on the low-level building blocks of Makie and don't get too focused on high-level API questions. In my opinion the foundation needs to be really stable and powerful before we can build on top of that. Right now there are still many small issues, for example with buffers for multiple attributes, or text display, or clipping, or composition of multiple plot objects, and so on. These should ideally be solved before writing recipes unless all the recipe work should be redone at a later point when low-level things change..

**Comment by SimonDanisch** [comment]
>To add to this, I think it's best if we concentrate on the low-level building blocks of Makie and don't get too focused on high-level API questions. In my opinion the foundation needs to be really stable and powerful before we can build on top of that. Right now there are still many small issues, for example with buffers for multiple attributes, or text display, or clipping, or composition of multiple plot objects, and so on. These should ideally be solved before writing recipes unless all the recipe work should be redone at a later point when low-level things change..

I'd like to second that...Last vizcon I think we (I) spent too much time on discussions that were pretty irrelevant to the concrete things that need to be done now - this time i'd personally would like to spent more time to get things done...
Of course, people who want to discuss bigger, theoretical things, can still do this in a group :)

**Comment by asinghvi17** [comment]
I'd like to give a small (10 min or less) talk about MakieGallery and its architecture, and solicit some suggestions for improvements, if we have the time for that.

**Comment by mkborregaard** [comment]
@SimonDanisch so do you suggest we remove the "group discussions" and move them to smaller groups?
Great idea to focus on internals, but can 21 people do that at the same time? Maybe better to have "internals team" and "interface team" etc?

**Comment by c42f** [comment]
> My goal is to establish a stronger foundation with scene clipping and axes

On a very nuts-and-bolts level, I'd be interested in the various aspects of making clipping work properly including the GPU and shader side of things (assuming that's still an open problem).

**Comment by asinghvi17** [comment]
It still is, to the best of my knowledge.

**Comment by SimonDanisch** [comment]
Well, scenes now rectangularly clip their content, and @jkrumbiegel uses this in MakieLayout to do correct clipping..but he got greedy and now also wants circular clipping etc :D

**Comment by asinghvi17** [comment]
Well it's not just him ;) - arbitrary clipping would be needed for earth plots!

**Comment by strangeli** [comment]
Hi, 
I am kind of back to that type of life again which does include communication, be easy on me though... I was never good at short texts.

Here is what I felt responsible for so far w.r.t organization: 

1. The secretary will give the attendees list to the door people and I will update it on Friday, Mach 13.
 
2. I will not be in Berlin on Thursday, March 12. Simon D will organize the keys for the first day (if you want to open/lock the room etc), from Friday on we can use my key. 

3. If needed, we can use up to 5 rooms on Sat/Sun - that is where my key fits and no one is here anyway. The other days except for Thursday (i.e., Fri/Mon), we could co-use some of those rooms, too.

4. I ordered 25 guest accounts for internet access in case people have no eduroam.

5. Our technical assistant is on vacation. I may ask her next Monday if she has equipment for streaming if this is still needed. (I found a Canon 600D so far but it seems not suitable for streaming over long periods of time.) According to what I read here, we do have a cam, a mic, that is enough?!

6. Restaurants (I don't care, just because I was asked): My favourite around here is currently only open till 6pm (weird...) [1], then there is a very close and funny restaurant on a ship [2] and on my beloved long and lonely evenings in the office I order food or go to [3], [4] by bike and there is a lot of food in Moabit (near Turmstraße) not so far away. Hence, I will give you my customized suggestions on Friday Mar 13 depending on your willingness for mobility and adventure. 

7. And for drinking - this is like a really traditional bar in that area: http://www.zwiebelfisch-berlin.de/, https://goo.gl/maps/vhdnhvaWCdYYqvu57

...? Lia

[1] Used to be a beer garden: http://www.schleusenkrug.de/contact.html
[2] Ship: https://www.capt-schillow.de/hier-finden-sie-uns/
[3] Vegan/vegetarian Indian restaurant: http://www.mysatyam.de/anfahrt.html
[4] Vietnamese/sushi around Savignyplatz - seems that I can't find it online. It is either closed or it is so cool that it has no google maps. Here is is still as Lemongrass https://www.openstreetmap.org/node/26735763#map=18/52.50736/13.32075 with a non-working website link. Anyway, I might find out. I always go to that corner and there is some Asian restaurant changing names once in a while.

**Comment by asinghvi17** [comment]
Thanks for the info, Lia!  I had one other question - does the seminar room come with chalk included?  I'm bringing along 8 sticks of Hagoromo, was wondering if that would be sufficient (I have had pretty bad experiences with the standard chalk brands, at least at my school 😄)

**Comment by mkborregaard** [comment]
OK everyone, the feedback on the program can be summarized with Elvis' undying words: "A little less conversation, a little more action". So I've taken out the discussions, and placed most of the presentations at the very beginning, where they should serve to get everyone on the same page for the most efficient collaboration. Again - please comment, this is just another rough scetch. Also, if you're mentioned by name in the schedule, please make sure you're happy to do what it says :-)
![Skærmbillede 2020-03-06 kl  08 49 50](https://user-images.githubusercontent.com/8429802/76063362-d95a0300-5f87-11ea-8eac-6326061bec20.png)

**Comment by jkrumbiegel** [comment]
Ideally I would want to show MakieLayout before the group discussion on Makie recipes. It will probably change people's ideas a bit how recipes should be implemented compared to the single subplot architecture we have now.

**Comment by mkborregaard** [comment]
I think that would be best, but you'll not be there before Friday afternoon, right? I think we need the recipes discussion at the very beginning. FWIW RecipesBase recipes don't actually really specify layouts, so for compatibility it's not really neaede.

**Comment by jkrumbiegel** [comment]
> FWIW RecipesBase recipes don't actually really specify layouts, so for compatibility it's not really neaede

True, although for any new system they will, so the talk should be early, but not necessarily in front of the RecipesBase talk. But StatsMakie and Theming might be better afterwards.

And yes I'll be there for the start, for all who didn't get the PM :)

**Comment by j-fu** [comment]
Hi, in the sense of "talk first to collect the ideas, then work"  you can reschedule my Sunday talk to any earlier slot, as the event I was supposed to participate in on the 12/13 fell victim to the virus...

**Comment by mkborregaard** [comment]
OK, I edited the schedule in the post above.

**Comment by mkborregaard** [comment]
Everyone - please remember to consider work themes or tasks you'd like to engage in. We'll still have discussions, but many of these will be in interest groups, to give the overall group as much flexibility as possible. Also, if you have an idea you'd like to pitch to the group - let us know if you'd like 5 minutes to show some slides and ideas!

**Comment by karlwessel** [comment]
My personal goal for the con is to get some insights on how backend implementation in Makie and Plots work and if it is possible to get Makie working with older OpenGL versions. 

However, even if I couldn't learn anything about that during the con (which I strongly doubt), I consider this as a good occasion for concentrated hacking to improve this great visualisation ecosystem. For example by fixing issues of Plots.jl or improving the very promising backend for PGFPlotsX.

**Comment by strangeli** [comment]
> Thanks for the info, Lia! I had one other question - does the seminar room come with chalk included? I'm bringing along 8 sticks of Hagoromo, was wondering if that would be sufficient (I have had pretty bad experiences with the standard chalk brands, at least at my school smile)

Sorry for the late reply, there is always enough chalk in the room and I never had problems but feel free to bring your own. 

Can anyone say again what is missing for this streaming thing? Our technical assistant does not know either and the online lectures are not streamed but recorded and uploaded such that I still don't know what streaming involves. So I heard someone of you is bringing a camera and we go with that, right?

As alternative, there seems to be a video conferencing room in the same building, ground floor (E-N 053). I asked for access and just got the immediate reply that we can use it from March 13-16 (Thursday is occupied). I am in home office today so I can check it out on Monday with the technical support person. It has a system which seems to make it easy to access remotely (without special software I was told so far) but the number of external attendees might be limited. I will keep you updated on the technical details. It is for free and nothing happens if we don't use it.

Btw, they canceled our German research meeting due to Corona, such that I will be there on Thursday, too.

**Comment by strangeli** [comment]
Ok, no the video conferencing needs a video conferencing equipment on the other side but we can use a digital meeting room of the German research network (DFN) in the normal seminar room. I created a vizcon 2 room already and can share the login details for everyone on Thursday (how and where to post the PIN, in slack?) and in addition, there is a streaming ("read-only") option for non-participants with chat option if wanted. Ok, now I feel better because I know how it could work - in theory :-). 

We still need a webcam though. I count on whoever said will bring one.

**Comment by asinghvi17** [comment]
AFAIU, @logankilpatrick is bringing a camera.  Slack would probably be best for the PIN, since it won't vanish until after the con (I think).  Thanks for the setup!

**Comment by logankilpatrick** [comment]
I’m just bringing my iPhone 11 Pro which has a great camera along with a mic to record the speakers audio.

**Comment by pfitzseb** [comment]
I can also bring a camera if necessary.

**Comment by strangeli** [comment]
Cool, this is the docs for possible layouts if we use the DFN meeting room if anyone wants to checkout beforehand what seems suitable: https://docs.pexip.com/admin/changing_layout.htm

**Comment by asinghvi17** [comment]
This is the link to what will be (hopefully) the live stream of Vizcon!  https://www.conf.dfn.de/stream/pfq8gtqq5mc

**Comment by glennmoy** [comment]
Hey guys, looking forward to meeting you all. I've dipped my toes into some plotting functionality here and there but I'm hoping to learn a few new things at vizcon and contribute what I can. See you in berlin!

**Comment by j-fu** [comment]
Bad news:

https://www.pressestelle.tu-berlin.de/menue/tub_medien/newsportal/innenansichten/2020/coronavirus_2019_ncov/sofortmassnahmen_fuer_alle_berliner_hochschulen_und_universitaeten :

Berlin universities (and also my insitute) cancel all events due to the virus...
